drone/containers/0.5/drone-debug/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:14.04
+MAINTAINER Nathan Valentine < nathan@rancher.com | nrvale0@gmail.com >
+
+ENV DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
+
+RUN apt-get update && apt-get dist-upgrade -y && \
+	rm -rf /var/cache/apt/archive
+RUN apt-get install -y mysql-client nmap python-pip curl wget vim bash && \
+	rm -rf /var/cache/apt/archive
```

Comment:
@cloudnautique 

When doing builds I noticed that it shrank the individual layers relative to not purging /var/cache/apt/archive on each line. Just a minor optimization.